pkg/execute/plugin_executor.go

```diff
@@ -234,3 +248,39 @@ func (e *PluginExecutor) getEnabledPlugins(bindings []string, cmdName string) ([
 
 	return out, fullPluginName
 }
+
+func (e *PluginExecutor) generateKubeConfig(rbac config.PolicyRule) (string, error) {
```

Comment:
Remember that this kubeconfig should be deleted after execution. And yes, for now we should support:

- static mapping for both users and groups
- channel-based group mapping.

Reply:
discussed f2f, for now we can don't remove them as it may generates additional problems, e.g.:
- multiple users call the executor (create dedicated files? make sure that we use the same one?)
- clean-up logic: can we remove file? what if it's still in use by other executor call?

so maybe we can skip it to just simplify the first implementation and create a follow-up task that we can refine and impl later if needed.